src/client.toit

```diff
@@ -105,6 +105,14 @@ class Client:
           logger_.warn "unpaired response"
     finally:
       close
+  
+  /**
+  Utility function to fetch a REAL/FLOAT from holding registers
+  */
+  read_float start_address/int -> float:
```

Comment:
I wonder if calling it `read_float32` would make more sense, as `float` in Toit is 64 bit.